src/pages/sponsor.module.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  margin: 0 auto;
+  padding: 24px 24px;
+  max-width: 960px;
+  background-color: #121212;
+  color: white;
+  align-items: center;
+}
+
+.dispImage {
+  object-fit: contain;
+  object-position: center;
+  width: 100%;
+  max-width: 500px;
+  height: auto;
+}
+
+.h2alt {
```

Comment:
CSS classes should be named descriptively so someone new to the codebase can quickly and easily understand what it's for. If a class is specific to a layout problem you encountered with a particular component, name it after that component (e.g. `.heading-your-support`).
CSS comments are also a good idea for one-off stuff :)